oscn/parse/attorneys.py

```diff
@@ -0,0 +1,20 @@
+from bs4 import BeautifulSoup
+
+
+def attorneys(oscn_html):
+    attorney_list = []
+    soup = BeautifulSoup(oscn_html, 'html.parser')
+    start = soup.find('h2', 'section attorneys')
+    sibling = start.next_sibling.next_sibling
```

Comment:
I did this first and it would crash if there weren't any attorneys. I will see if I can run the if with teh return from find_next_sibling. That might be prettier.

Reply:
Yes that works. updated.